### ISSUE TITLE
More support for Progressive Web Apps (PWA's) and shortcuts.

### ISSUE THREAD

**Issue by leslie-kretai** [8bitPit/Niagara-Issues#2395]
### Is your feature request related to a problem?

When "installing" PWA's or adding shortcuts from the browser they automatically get added to your favourites but not the main app list. Removing them from favourites results in complete deletion rather than continuing to list it in the main list.

Issue noticed while using Vanadium Browser, same issue observed in Brave Browser and Firefox.

### Describe the solution you'd like

Add more specific functionality to handle browser shortcuts and PWA's; so that they may be displayed and organized as if they were a native android app.

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by ssssamstrup** [comment]
Doesn't this work now? Or is it only for url-shortcuts?

**Comment by GlitchWalker** [comment]
I don't believe this issue has been completed? Came here to file it as a feature request. PWAs installed via any browser other than Google Chrome still only appear in favourites, and when you delete the favourite the app is deleted.

I understand why this is, Android (still) only supports PWAs installed via Chrome, despite apparently being on their roadmap. But I was really hoping Niagara could support PWAs from other browsers as if they were native.

Currently on app version 1.13.3, Android version Baklava on Pixel 7a, but behaviour also happens on my Android 13 Umidigi tablet.

**Comment by lesliek-dev** [comment]
I closed the issue because I noticed that they added functionality to support the specific flow I was looking for (Installing a PWA from "Chrome"; in this case Vanadium, and displaying it as a regular app). I was quite busy at the time so I apologize for the lack of follow up. It seems like some people are still having some issues with specific browsers or the flow not being as ergonomic/intuitive as it could be.

I can reopen this issue if people would like, however it is a rather broad issue so I feel those situations would be better represented by their own issue in reference to this one and #2018.
